packages/regl-worldview/src/commands/Command.js

```diff
@@ -20,6 +20,7 @@ export type Props<T> = {
   reglCommand: RawCommand<T>,
   getHitmapId?: (T) => ?number,
   layerIndex?: number,
+  [MouseEventEnum]: ComponentMouseHandler,
```

Comment:
Shouldn't this be optional? Why doesn't Flow complain about it being missing?

Reply:
it's implicitly optional